libcontainer/cgroups/utils.go

```diff
@@ -277,6 +277,7 @@ func RemovePaths(paths map[string]string) (err error) {
 			}
 		}
 		if len(paths) == 0 {
+			//nolint:ineffassign // done to help garbage collecting: opencontainers/runc#2506
```

Comment:
Looks like it at least finds them 😂 had a typo here;

```
level=warning msg="[runner/nolint] Found unknown linters in //nolint directives: ineffasign"
```

Reply:
Can we change the function to return a separate map?

Reply:
> Can we change the function to return a separate map?

I don't think this makes any sense. No caller of RemovePaths is interested in this map; it is cleaned here in place only to help garbage collector.

If you don't like it we can just drop the `paths = make(map[string]string)` line, but I think it makes sense to keep it.